Examples/Io/Performance/ActsExamples/Io/Performance/SeedingPerformanceWriter.cpp

```diff
@@ -95,10 +97,20 @@ ActsExamples::ProcessCode ActsExamples::SeedingPerformanceWriter::finalize() {
       "/ nMatchedParticles) = "
       << aveNDuplicatedSeeds);
 
+  auto write_float = [&](float f, const char* name) {
+    TVectorF v(1);
+    v[0] = f;
```

Comment:
Does this not work with `std::vector<float>`?

Reply:
yes, it stores a `std::vector<float>`

Reply:
I meant, can't you just do 
```suggestion
    std::vector<float> v {f};
```

Reply:
If this works, you could maybe remove the includes!